operators/benches/cache_concurrent.rs

```diff
@@ -0,0 +1,240 @@
+use std::hint::black_box;
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::sync::Arc;
+
+use futures::future::join_all;
+use geoengine_datatypes::primitives::{DateTime, SpatialPartition2D, SpatialResolution};
+use geoengine_datatypes::raster::RasterProperties;
+use geoengine_datatypes::{
+    primitives::{RasterQueryRectangle, TimeInterval},
+    raster::{Grid, RasterTile2D},
+    util::test::TestDefault,
+};
+use geoengine_operators::{engine::CanonicOperatorName, pro::cache::tile_cache::TileCache};
+use lazy_static::lazy_static;
+use rand::rngs::SmallRng;
+use rand::{Rng, SeedableRng};
+use serde_json::json;
+
+lazy_static! {
+    static ref WRITTEN_ELEMENTS: AtomicUsize = AtomicUsize::new(0);
+}
```

Comment:
Now you really got me confused that you used lazy static for initializing and using the `AtomicUsize`.

Just initialize it:

```Rust
use std::sync::atomic::{AtomicUsize, Ordering};

static GLOBAL_VALUE: AtomicUsize = AtomicUsize::new(0);

fn main() {
    dbg!(GLOBAL_VALUE.load(Ordering::Relaxed));
    
    GLOBAL_VALUE.fetch_add(1, Ordering::Relaxed);
    
    dbg!(GLOBAL_VALUE.load(Ordering::Relaxed));
    
    GLOBAL_VALUE.fetch_add(1, Ordering::Relaxed);
    
    dbg!(GLOBAL_VALUE.load(Ordering::Relaxed));
    
    GLOBAL_VALUE.fetch_add(1, Ordering::Relaxed);
    
    dbg!(GLOBAL_VALUE.load(Ordering::Relaxed));
}
``

Reply:
c2045f72e7646d62e97d8dcca65602d66b58d591